packages/node_modules/@node-red/editor-client/src/js/nodes.js

```diff
@@ -568,8 +568,24 @@ RED.nodes = (function() {
         return node;
     }
 
+    function collectNodesInSubflow(n) {
+        var nodes = RED.nodes.filterNodes({z:n.id});
+        return createExportableNodeSet(nodes);
+    }
+    
     function convertSubflow(n) {
         var node = {};
+        var id = n.id;
+        if (n.flow) {
```

Comment:
Please move this to below where `node.name` is set on line 591. This will mean the `id`,`type` and `name` properties will appear first in the generated JSON. Also, move setting `node.flow` to the end (after setting `node.description`. This makes it easier to look at the JSON and read off those properties before getting lost in the `flow` array of nodes.

Reply:
fixed in phase3 implementation.